src/Client.js

```diff
@@ -1739,6 +1739,24 @@ class Client extends EventEmitter {
         }
         return 0;
     }
+
+    /**
+     * Sync chat history conversation
+     * @param {string} chatId
+     * @return {Promise<boolean>} True if operation completed successfully, false otherwise.
+     */
+    async syncHistory(chatId) {
+        return this.pupPage.evaluate(async (chatId) => {
```

Comment:
Why there is not await?

Reply:
@shirser121 
`async syncHistory(chatId)`, this aready returns `Promise`, you then call it like that:
```js
await syncHistory(...)
```